nexus/catalog/migrations/V39__table_schema_mapping.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS table_schema_mapping (
+    flow_name varchar(255) not null,
```

Comment:
I originally wanted it to be a foreign key on flows which has same constraint, but FK doesn't work because no unique constraint because of that weird way we store multiple table mappings. So this can be changed when we clean that up